test/integration/utils/utils.go

```diff
@@ -32,3 +35,19 @@ func GetTestProjectFromSetup(t *testing.T, idx int) string {
 	}
 	return projectIDs[idx]
 }
+
+var (
+	// TGKEVerify Exempt Resources. e.g. google_container_cluster.primary
+	verifyExemptResources = []string{}
+)
+
+func TGKEVerify(t *testing.T, b *tft.TFBlueprintTest, assert *assert.Assertions) {
```

Comment:
Would it make sense to allow the call site to provide the slice of address to ignore? We can export `verifyExemptResources` as a starting point.

Reply:
Added `TGKEVerifyExemptResources()`, LMKWYT